cfme/cloud/provider.py

```diff
@@ -48,6 +48,7 @@
         ('ipaddress_text', Input("ipaddress"), {"removed_since": "5.4.0.0.15"}),
         ('region_select', {version.LOWEST: Select("select#provider_region"),
             "5.5": AngularSelect("provider_region")}),
+        ('amazon_region_select', AngularSelect("provider_region")),
```

Comment:
@psav I'm not sure if 'region_select' is still needed for 5.5, I just noticed that the 'amazon_region_select' is needed and is missing. Same goes for `properties_form_56`, there are both 'region_select' and 'amazon_region_select' with same value.